pages/singlesite/[slug].js

```diff
@@ -11,45 +16,234 @@ export default function singleSite() {
+                  className="ancient-picture"
+                  src={currentSite.ancientPicture[0]}
+                  alt={currentSite.adress}
+                  height={200}
+                  width={300}
+                />
+
+                <figcaption className="figcaption-ancient">
+                  {currentSite.ancientCaption[0]}
+                </figcaption>
+              </StyledFigure>
+            </StyledSwiperSlide>
+            <StyledSwiperSlide>
+              <StyledFigure>
+                <StyledImage
+                  className="ancient-picture"
+                  src={currentSite.ancientPicture[1]}
+                  alt={currentSite.adress}
+                  height={200}
+                  width={300}
+                />
+                <figcaption className="figcaption-ancient">
+                  {currentSite.ancientCaption[1]}
+                </figcaption>
+              </StyledFigure>
+            </StyledSwiperSlide>
+            <StyledSwiperSlide>
+              <StyledFigure>
+                <StyledImage
+                  className="ancient-picture"
+                  src={currentSite.ancientPicture[2]}
+                  alt={currentSite.adress}
+                  height={200}
+                  width={300}
+                />
+                <figcaption className="figcaption-ancient">
+                  {currentSite.ancientCaption[2]}
+                </figcaption>
+              </StyledFigure>
+            </StyledSwiperSlide>
+            <StyledSwiperSlide>
+              <StyledFigure>
+                <StyledImage
+                  className="ancient-picture"
+                  src={currentSite.ancientPicture[3]}
+                  alt={currentSite.adress}
+                  height={200}
+                  width={300}
+                />
+                <figcaption className="figcaption-ancient">
+                  {currentSite.ancientCaption[3]}
+                </figcaption>
+              </StyledFigure>
+            </StyledSwiperSlide>
+            <StyledSwiperSlide>
+              <StyledFigure>
+                <StyledImage
+                  className="ancient-picture"
+                  src={currentSite.ancientPicture[4]}
+                  alt={currentSite.adress}
+                  height={200}
+                  width={300}
+                />
+                <figcaption className="figcaption-ancient">
+                  {currentSite.ancientCaption[4]}
+                </figcaption>
+              </StyledFigure>
+            </StyledSwiperSlide>
+            <StyledSwiperSlide>
+              <div className="credits-card">
+                <h2>
+                  {currentSite.creditsCaption + ": "}
+                  {currentSite.coordinates[0] +
+                    " | " +
+                    currentSite.coordinates[1]}
+                </h2>
+                <p className="credits">
+                  {currentSite.credits[0]}
+                  <br />
```

Comment:
try not to use </br>. Style it with CSS ;)

Reply:
Will try! (You are talking about the <br>, right?)